prbt_hardware_support/test/unittests/unittest_modbus_api_spec.cpp

```diff
@@ -111,6 +111,5 @@ TEST(ModbusApiSpecTest, NodeHandleConstructionMissingApiSpec)
 
 int main(int argc, char **argv){
   testing::InitGoogleTest(&argc, argv);
-  ros::init(argc, argv, "unittest_modbus_api_spec");
```

Comment:
Is this related to the intention of this PR?

Reply:
No